src/App.tsx

```diff
@@ -412,9 +429,10 @@ function App(): ReactElement {
       }
       setFeatureName(newFeatureName);
 
-      if (!isColorRampRangeLocked) {
-        setColorRampMin(newDataset?.features[newFeatureName].min || colorRampMin);
-        setColorRampMax(newDataset?.features[newFeatureName].max || colorRampMax);
+      const featureData = newDataset.getFeatureData(newFeatureName);
+      if (!isColorRampRangeLocked && featureData) {
+        setColorRampMin(featureData.min);
+        setColorRampMax(featureData.max);
```

Comment:
Before, datasets with a min feature value of 0 would not update the `colorRampMin` (because 0 is falsy).